src/tests/unit/tests/DevTools/target-page-inspector.test.ts

```diff
@@ -61,9 +61,6 @@ describe('TargetPageInspector', () => {
         actualSelector => {
             // we need to define a inspect function so we can actually evaluate the script
             // tslint:disable-next-line: no-unused-variable
-            function inspect(): void {
```

Comment:
This is an intentional/required unused local (see the comment above it and the unit test failures) - we'll want to replace the tslint suppression line above it with `// @ts-expect-error` instead